### PR TITLE
Your single sample analysis outputs

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Refactored `Your Single Sample Analysis Outputs` page to use Ant Design.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [X] Tests added (or description of how to test) for any new features.
* [X] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java [hunk 1]
```diff
@@ -121,19 +121,6 @@ public AnalysisAjaxController(AnalysisSubmissionService analysisSubmissionServic
...
-	/**
-	 * Get all {@link User} generated {@link AnalysisOutputFile} info for principal User
-	 *
-	 * @param principal Principal {@link User}
-	 * @return {@link User} generated {@link AnalysisOutputFile} info
-	 */
-	@RequestMapping(value = "/user/analysis-outputs")
-	@ResponseBody
-	public List<ProjectSampleAnalysisOutputInfo> getAllUserAnalysisOutputInfo(Principal principal) {
-		final User user = userService.getUserByUsername(principal.getName());
```
Comment (joshsadam) [inline src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java:133]: Don't get the user here, do this is the service.  This constructor is out of control (and I know that is not your fault).  You can use the `SecurityContextHolder` to get the current user from within the service.

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+	}
+
+	/**
+	 * Get all the automated single sample analysis outputs for the project
+	 *
+	 * @param principal Currently logged in user.
+	 * @return a list of filtered {@link ProjectSampleAnalysisOutputInfo} single sample analysis outputs for the user
+	 */
+	public List<ProjectSampleAnalysisOutputInfo> getUserSingleSampleOutputs(Principal principal) {
+		User user = userService.getUserByUsername(principal.getName());
```
Comment (joshsadam) [inline src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java:83]: As I said above, you can get the user using `SecurityContextHolder`, and therefore don't really need to pass it into here.

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+	/**
+	 * Prepare the download of multiple {@link AnalysisOutputFile} by adding them to a selection.
+	 *
+	 * @param outputs  Info for {@link AnalysisOutputFile} to download
+	 * @param response {@link HttpServletResponse}
+	 */
+	public void prepareAnalysisOutputsSelectionDownload(List<ProjectSampleAnalysisOutputInfo> outputs,
+			HttpServletResponse response) {
+		analysisOutputFileDownloadManager.setSelection(outputs);
+		response.setStatus(HttpServletResponse.SC_CREATED);
```
Comment (joshsadam) [inline src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java:105]: Why are you setting the status of the response in the service?  This is really one of the only things that should be done in the controller.  If the service does not fail then set a `ResponseEntity` with the status of created.

File: src/main/webapp/resources/js/components/analyses-outputs/SingleSampleAnalysisOutputs.jsx [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
-import { dateColumnFormat } from "../../../../components/ant.design/table-renderers";
-import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { dateColumnFormat } from "../ant.design/table-renderers";
+import { setBaseUrl } from "../../utilities/url-utilities";
 import {
   downloadIndividualOutputFile,
   downloadSelectedOutputFiles,
   prepareAnalysisOutputsDownload,
-} from "../../../../apis/projects/analyses";
+} from "../../apis/analyses/analyses";
```
Comment (joshsadam) [inline src/main/webapp/resources/js/components/analyses-outputs/SingleSampleAnalysisOutputs.jsx:12]: Internationalization on line 118 please

**Review by joshsadam — CHANGES_REQUESTED** [review]
Just a couple small things, but overall this looks really great.  Feel free to chat with me about any of it.

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java (hunk 1 above)
Comment (deepsidhu85) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java:133]: This looks like some code I forgot to remove as its been moved to the AnalysesOutputsAjaxController

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java (hunk 3 above)
Comment (deepsidhu85) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java:105]: This was something that was there before and I just reused. I can update it :)

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java (hunk 1 above)
Comment (deepsidhu85) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java:133]: Method removed in c6f2f6cfa2f5b74494bf05a58d24fd7fb08c35fb

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java (hunk 2 above)
Comment (deepsidhu85) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java]: Updated in c6f2f6cfa2f5b74494bf05a58d24fd7fb08c35fb

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java (hunk 3 above)
Comment (deepsidhu85) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnalysesOutputsService.java]: Updated in c6f2f6cfa2f5b74494bf05a58d24fd7fb08c35fb

File: src/main/webapp/resources/js/components/analyses-outputs/SingleSampleAnalysisOutputs.jsx (hunk 4 above)
Comment (deepsidhu85) [reply src/main/webapp/resources/js/components/analyses-outputs/SingleSampleAnalysisOutputs.jsx:12]: Added translation in c6f2f6cfa2f5b74494bf05a58d24fd7fb08c35fb

**Comment by joshsadam** [comment]
These tables run very nicely.  Let me know when it is all GTG

File: doc/user/user/pipelines/index.md [hunk 5]
```diff
@@ -278,17 +278,14 @@ You will see a table of your single sample analysis output files:
...
-![](images/batch-download-column-filter-open.png)
-
-Filtering for files with `contigs` in the filename:
-
-![](images/batch-download-filtering-on-contains-contigs.png)
-
-You will then see a filter icon displayed beside the column name ![](images/batch-download-column-filter-icon.png)
-
-You can select the files you wish to download using the checkboxes or by holding the `Ctrl` or `Shift` keys as you click on the rows for the files you wish to download. 
+You can select the files you wish to download using the checkboxes.
```
Comment (joshsadam) [inline doc/user/user/pipelines/index.md:288]: Do you actually mean download not delete?  I cannot download multiple at one time.

**Review by joshsadam — APPROVED** [review]

**Pull request merged**